frontend/help-chef/src/app/app.module.ts

```diff
@@ -1,13 +1,15 @@
 import { NgModule } from '@angular/core';
 import { BrowserModule } from '@angular/platform-browser';
+import { ReactiveFormsModule } from '@angular/forms';
 
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
-import { MainComponent } from './app/main/main.component';
```

Comment:
One mistake and localhost isn't working ;)

Reply:
Great that you found the solution for the issue :D